### ISSUE TITLE
[FP] replace empty() with `=== null`, where multiple parts may be null

### ISSUE THREAD

**Issue by puggan** [kalessil/phpinspectionsea#1157]
| Subject        | Details |
| :------------- | :---------------------------------------------------------------------------- |
| Issue type     | False-Positive 
| Plugin         | Php Inspections (EA Ultimate) v2.0.11 |
| Language level | PHP 5.0 - 7.3, see https://3v4l.org/TsUuU |

Current behaviour (description/screenshot:)
--
Sugesting to replace `empty()` with `=== null`, when more then one of the properties may be null.
```php
<?php

/**
 * Class A
 */
class A {
    /** @var B|null */
    public $b;
}

/**
 * Class B
 */
class B {
    /** @var C|null */
    public $c;
}

/**
 * Class C
 */
class C {
    /** @var string|null */
    public $d;
}

$a = new A();

if(empty($a->b->c->d)) echo $a->b->c->d;
```

![screenshot from 2019-01-07 13-10-24](https://user-images.githubusercontent.com/866668/50767669-f9a40a00-127d-11e9-9b8f-e8c3d66adcd6.png)

Expected behaviour
--
As $a->b, $a->b->c or $a->b->c->d may be null, you can't replace it with comparing just one of them.

https://3v4l.org/TsUuU

Environment details
---
PhpStorm 2018.3.2
Build #PS-183.4886.22, built on December 12, 2018
JRE: 1.8.0_152-release-1343-b26 amd64
JVM: OpenJDK 64-Bit Server VM by JetBrains s.r.o
Linux 4.18.0-13-generic

**Comment by kalessil** [comment]
Confirmed, thank you for reporting @puggan!

**Comment by kalessil** [comment]
Fixed!
